lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -300,9 +305,18 @@ class RenderPopover extends RenderShiftedBox {
 
     context.canvas.drawPath(borderPath.shift(offset), _backgroundPaint);
 
-    if (child != null) {
-      context.paintChild(child!, offset + _contentOffset);
-    }
+    _clipPathLayer.layer = context.pushClipPath(
+      needsCompositing,
+      offset,
+      Offset.zero & child!.size,
+      borderPath,
+      (PaintingContext innerContext, Offset innerOffset) {
+        if (child != null) {
```

Comment:
You've already referenced `child!` up above, so either we already know `child` is non-null, or we've already crashed. If a `null` `child` is a possibility then that needs to be handled before this point. But what would it mean to have a `null` `child`? Is that possible?